CONTRIBUTING.md

```diff
@@ -77,14 +118,14 @@ schemas\Validate.ps1
 In accordance with Section "Recommendation for Commit Messages" the continuous integration checks the previously defined conditions.
 For the present development, however, this is not enforced.
 
-## Approval Process
+## Approval Process for PR
 All changes must be **reviewed** and **approved**.
 
 Minor changes (simple failures, typos, *etc.*) and additional content (more examples, etc.) can be accepted straight away after a brief review by the responsible reviewers.
 
-Major changes must first be reviewed and approved by the joint working group "Asset Administration Shell" of the [Platform Industrie 4.0] and [IDTA].
+Major changes (i.e. pull requests associated to issues labels classified as "major") must first be reviewed and approved by the Workstream "Specification Asset Administration Shell" within the "OpenTechnology" Working "Asset Administration Shell" of the  [IDTA].
```

Comment:
same as above:
"... within the "OpenTechnology" Working "Asset Administration Shell" of the  [IDTA]." --> "... within the "OpenTechnology" Working Group of the [IDTA]."